packages/wrangler/src/pages/functions/buildWorker.ts

```diff
@@ -222,6 +222,7 @@ export function buildRawWorker({
 			targetConsumer: local ? "dev" : "publish",
 			local,
 			experimentalLocal: false,
+			forPages: true,
```

Comment:
Is this the only place Pages calls `bundleWorker`?

Reply:
no sorry, I missed a couple of places 😓, I've added the option there as well 🙏